ios/nevis_mobile_authentication_sdk_example_app_react.xcodeproj/project.pbxproj

```diff
@@ -681,7 +688,10 @@
 					"\"$(inherited)\"",
 				);
 				MTL_ENABLE_DEBUG_INFO = NO;
-				OTHER_CFLAGS = "$(inherited)  ";
+				OTHER_CFLAGS = (
+					"$(inherited)",
+					" ",
+				);
```

Comment:
At some point we should figure out why is this changing all the time back-and-forth...